chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/RoomService.kt

```diff
@@ -1,125 +1,117 @@
-                        joinedRoomApiMapper.toRoom(response)
-                    }
+    ) {}
+//            : Result<Room, Error> =
+//            CreateRoomRequest(
+//                    id = id,
+//                    name = name,
+//                    pushNotificationTitleOverride = pushNotificationTitleOverride,
+//                    private = isPrivate,
+//                    createdById = creatorId,
+//                    customData = mapOf<String, Any>(),
+//                    userIds = userIds
+//            ).toJson()
+//                    .flatMap { body -> client.doPost<CreateRoomResponse>("/rooms", body) }
+//                    .map { response ->
+//                        roomStore.add(response)
+//                        userService.populateUserStore(response.membership.userIds.toSet())
+//                        joinedRoomApiMapper.toRoom(response)
+//                    }
 
     fun deleteRoom(roomId: String): Result<String, Error> =
             legacyV2client.doDelete<Unit?>("/rooms/${URLEncoder.encode(roomId, "UTF-8")}")
                     .map {
-                        roomStore.remove(roomId)
+                        //                        roomStore.remove(roomId)
                         roomId
                     }
 
     fun leaveRoom(userId: String, roomId: String): Result<String, Error> =
             client.doPost<Unit?>("/users/${URLEncoder.encode(userId, "UTF-8")}/rooms/${URLEncoder.encode(roomId, "UTF-8")}/leave")
                     .map {
-                        roomStore.remove(roomId)
+                        //                        roomStore.remove(roomId)
                         roomId
                     }
 
-    fun joinRoom(userId: String, roomId: String): Result<Room, Error> =
-            client.doPost<JoinRoomResponse>("/users/${URLEncoder.encode(userId, "UTF-8")}/rooms/${URLEncoder.encode(roomId, "UTF-8")}/join")
-                    .map { response ->
-                        roomStore.add(response)
-                        userService.populateUserStore(response.membership.userIds.toSet())
-                        joinedRoomApiMapper.toRoom(response)
-                    }
+    fun joinRoom(userId: String, roomId: String) {}
+//            : Result<Room, Error> =
+//            client.doPost<JoinRoomResponse>("/users/${URLEncoder.encode(userId, "UTF-8")}/rooms/${URLEncoder.encode(roomId, "UTF-8")}/join")
+//                    .map { response ->
+// //                        roomStore.add(response)
+//                        userService.populateUserStore(response.membership.userIds.toSet())
+//                        joinedRoomApiMapper.toRoom(response)
+//                    }
 
     fun updateRoom(
         roomId: String,
         name: String? = null,
         pushNotificationTitleOverride: RoomPushNotificationTitle? = null,
         isPrivate: Boolean? = null,
-        customData: CustomData? = null
+        customData: Map<String, Any>? = null
```

Comment:
we could keep CustomData as typealias here

Reply:
RoomService has now been deleted, so this change is no longer necessary.